pages/mesosphere/dcos/services/hdfs/2.6.0-3.2.0/release-notes/index.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+excerpt: Discover the new features, updates, and known limitations in this release of the HDFS Service
+title: Release Notes
+menuWeight: 0
+model: /mesosphere/dcos/services/hdfs/data.yml
+render: mustache
+---
+
+# Release Notes for HDFS Service version 2.6.0-3.2.0
+
+## Feature
+- HDFS cluster can be accessed from outside the DC/OS cluster by setting up an external DNS. This feature can be utilized through `custom_domain` property.
+
+## Bug Fixes
+- A bug in Apache Derby has been fixed here temporarily. Bug referred to miscalculation of number of bytes read for an principal entry in the Kerberos keytab file. This is addressed in this PR: ([#44](https://github.com/apache/directory-kerby/pull/44)). There is also a ticket in Apache Hadoop repository: ([#16283](https://issues.apache.org/jira/browse/HADOOP-16283)).
+
+## Improvements
+- rollingUpgrade feature of Hdfs is used to upgrade the framework to latest Apache HDFS version.
+
+## Updates
+- Hadoop tar ball has been updated with snappy library.
```

Comment:
Please move this to the above Updates section including the version of snappy.

Reply:
Done